package.json

```diff
@@ -18,7 +18,7 @@
     "doc": "lib/docs"
   },
   "scripts": {
-    "postinstall": "node common/scripts/install-run-rush.js update",
```

Comment:
This gets shipped with the npmjs package -- which causes a failure for people when they try and install the package